cypress/integration/genome.js

```diff
@@ -23,11 +23,13 @@ describe('Genome page', () => {
                   .each(($row, idx) => {
                     $row.children('.column').each((_, el) => {
                         const $el = Cypress.$(el);
+                        const text = $el.text().replace(/\s+/g, ' ').replaceAll('\n', '');
```

Comment:
Just curious... isn't `\n` match for `/\s+/g` making the `replaceAll` redundant?

Reply:
You are right.... that is a "trying to make the test pass" kind of error.